### PR TITLE
Update infer_cli.py

### PR DESCRIPTION
**Pull request by thesandi99**
Save generated chunks in the output_dir folder; it helps with the multi-generation process.

### CODE REVIEW

**Comment by SWivid** [comment]
1. was saving to different path then the final one, better same path; and better named with same prefix as final one with postfix {chunk_num}_{chunk_gen_text} e.g.
2. not accurate annotation e.g. mkdir but annotated as save file
3. better introduce with a flag to control whether save chunks

**Comment by thesandi99** [comment]
🫡

File: src/f5_tts/infer/infer_cli.py [hunk 1]
```diff
@@ -207,16 +207,28 @@ def main_process(ref_audio, ref_text, text_gen, model_obj, mel_spec_type, remove
...
+        # Save entire wave file
         with open(wave_path, "wb") as f:
             sf.write(f.name, final_wave, final_sample_rate)
             # Remove silence
             if remove_silence:
                 remove_silence_for_generated_wav(f.name)
             print(f.name)
 
+        # Ensure the chunk directory exists
+        chunk_dir = "/content/aud/chunk"
```
Comment (ZhikangNiu) [inline src/f5_tts/infer/infer_cli.py:223]: os.path.join(output_dir,chunk_dir) will be a good choice

Comment (thesandi99) [reply src/f5_tts/infer/infer_cli.py:223]: actully im in running colab so ... its aceddently push

Comment (ZhikangNiu) [reply src/f5_tts/infer/infer_cli.py:223]: if you don't mind, I could add this pr in https://github.com/SWivid/F5-TTS/pull/518

Comment (thesandi99) [reply src/f5_tts/infer/infer_cli.py:223]: its work like save the chucks and if if len(audio) / final_sample_rate < 0.8 * ref_audio_length: #the output lenth is not match 80% of ref_audio is reganarate

Comment (thesandi99) [reply src/f5_tts/infer/infer_cli.py:223]: its more we can improve it like calculating the ref audios paramets and config the gen_audio like that

**Pull request closed without merging**